src/plotting/create_plots.py

```diff
@@ -78,10 +78,7 @@ def create_plots(args, root_path, use_datetime=True, test_private_keys: Optional
             pool_contract_puzzle_hash = decode_puzzle_hash(args.pool_contract_address)
 
     assert (pool_public_key is None) != (pool_contract_puzzle_hash is None)
-    if args.num is not None:
-        num = args.num
-    else:
-        num = 1
```

Comment:
Did you confirm that the underlying plotter will default to one plot absent this?

Reply:
Good point, from the block_tools is always set to 15 or 5, and from the cli is always set, because it has a default of 1 (from `click.Option` definition).